bases/br_geobr_mapas/mesorregiao/table_config.yaml

```diff
@@ -0,0 +1,169 @@
+# Quem está preenchendo esses metadados?
+published_by:
+    name: Gustavo Alcântara
+    email: gustavo.alcantara@basedosdados.org
+    github_user: gustavoalcantara
+    ckan_user:
+    website:
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+data_cleaned_by:
+    name: Rafael H.M Pereira
+    email: rafael.pereira@ipea.gov.br
+    github_user: rafaelpereirabr
+    ckan_user: 
+    website: https://github.com/ipeaGIT/geobr
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+data_cleaning_description: etl dos dados, rename e select para o Padrão da Base dos Dados.
+
+# Url do código de limpeza dos dados.
+data_cleaning_code_url: https://github.com/basedosdados/mais/tree/master/bases/br_geobr_mapas/code/mesorregioes
+
+# Organização que ajudou institucionalmente na disponibilização dos dados.
+partner_organization:
+    name:
+    organization_id:
+
+# Url dos dados originais no GCP Storage.
+raw_files_url:
+
+# Url dos arquivos auxiliares no GCP Storage.
+auxiliary_files_url:
+
+# Url da tabela de arquitetura no GCP Storage.
+architecture_url:
+
+source_bucket_name: basedosdados-dev
+
+project_id_prod: basedosdados-dev
+
+project_id_staging: basedosdados-dev
+
+# Liste as colunas da tabela que representam partições.
+# Não esqueça de deletar essas colunas nas tabelas .csv na hora de subir para o BigQuery.
+# Isso poupará muito tempo e dinheiro às pessoas utilizando essa tabela.
+# Se não houver partições, não modifique abaixo.
+partitions:
+
+# Quais são as colunas? Certifique-se de escrever uma boa descrição, as pessoas vão gostar
+# para saber sobre o que é a coluna.
+# Adicionar todas as colunas manualmente pode ser bastante cansativo, por isso, quando
+# inicializando este arquivo de configuração, você pode apontar a função para uma amostra de dados que
+# preencherá automaticamente as colunas.
+# Algumas colunas existirão apenas na tabela final, você as construirá em `publish.sql`.
+# Para esses, defina is_in_staging como False.
+# Além disso, você deve adicionar as colunas de partição aqui e definir is_partition como True.
+columns:
+    - name: id_uf
+      bigquery_type: string
+      description: ID da Unidade de Federação
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: uf
+          column_name: id_uf
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: sigla_uf
+      bigquery_type: string
+      description: Sigla da Unidade de Federação
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: uf
+          column_name: sigla_uf
```

Comment:
sigla

Reply:
sim

Reply:
Cris, mesma coisa do anterior, alterei pra não ter que alterar coluna no dataset

Reply:
ai ai ai vddddddddddd

Reply:
agora vai cris, por tudo que é sagrado

Reply:
Ô amigo, hahahahahahahahaha ... eu to rindo muito de ti. Mas ainda tá errado, tá? `column_name` = sigla, só **sigla** sem o `uf`. Lembra do Manual de estilo? Só usamos sufixos de entidade quando a entidade da coluna é diferente da entidade da tabela.

Reply:
HAHAHAHAHAHAHAHAHA dá um desconto que era sexta-feira  kkkk